grammars/gfm.cson

```diff
@@ -145,7 +145,7 @@
   }
   {
     'begin': '\\A---$'
-    'end': '^---$'
+    'end': '^---|\\.\\.\\.$' 
```

Comment:
Sorry, one last thing, I think this should be `^(---|\\.\\.\\.)$'  so that the lines terminator match both styles.